sunpy/util/scraper.py

```diff
@@ -334,7 +335,11 @@ def _ftpfileslist(self, timerange):
         ftpurl = urlsplit(directories[0]).netloc
         with FTP(ftpurl, user="anonymous", passwd="data@sunpy.org") as ftp:
             for directory in directories:
-                ftp.cwd(urlsplit(directory).path)
+                try:
+                    ftp.cwd(urlsplit(directory).path)
+                except Exception as e:
```

Comment:
So this is raised when we can't for whatever reason cd on the server? i.e. the directory doesn't exist? How did this work before hand??

Reply:
Yeah, so before in the client if you asked for 2040, if you didn't pre-restrict the time range given it would error here. Now it won't error if the directory is missing.

It was also why I added debugging to be sure.